backend/app/Actions/Visits/CreateVisitAction.php

```diff
@@ -88,6 +90,9 @@ public function execute(CreateVisitRequest $request): void
         $visit->geo_position_id = $geoPosition->id;
 
         $this->visitRepository->save($visit);
+
+        $user = $visit->visitor->website->user;
+        Notification::send($user, new NewVisitsNotification($visit));
```

Comment:
1) it is better to create event "VisitCreated" and dispatch it here with $visit. VisitCreated::dispatch($visit)
2) Then add event listener and register it. In your event listener send notification to private channel with websiteId. Because, if we will have teams, all team should see changes on widget, not only user that has website.

https://laravel.com/docs/5.8/events